NodeFirefox/Dockerfile

```diff
@@ -1,22 +1,24 @@
-FROM selenium/node-base:2.53.0
+FROM selenium/node-base:2.53.1
 MAINTAINER Selenium <selenium-developers@googlegroups.com>
 
 USER root
 
 #=========
 # Firefox
 #=========
-ENV FIREFOX_VERSION 45.0.2
 RUN apt-get update -qqy \
   && apt-get -qqy --no-install-recommends install firefox \
   && rm -rf /var/lib/apt/lists/* \
-  && wget --no-verbose -O /tmp/firefox.tar.bz2 https://download-installer.cdn.mozilla.net/pub/firefox/releases/$FIREFOX_VERSION/linux-x86_64/en-US/firefox-$FIREFOX_VERSION.tar.bz2 \
-  && apt-get -y purge firefox \
-  && rm -rf /opt/firefox \
-  && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
-  && rm /tmp/firefox.tar.bz2 \
-  && mv /opt/firefox /opt/firefox-$FIREFOX_VERSION \
-  && ln -fs /opt/firefox-$FIREFOX_VERSION/firefox /usr/bin/firefox
+  && apt-get -y purge firefox
+RUN wget --no-verbose -O /tmp/firefox-45.0.2.tar.bz2 https://download-installer.cdn.mozilla.net/pub/firefox/releases/45.0.2/linux-x86_64/en-US/firefox-45.0.2.tar.bz2 \
```

Comment:
If we want to have that, then all of the versions should be defined in an ENV variable, e.g., `FIREFOX_LATEST`, `FIREFOX_MINUS_ONE`, `FIREFOX_MINUS_TWO`. And installing these three versions should be done in a `for`-loop to make the Dockerfile more readable.